aspnetcore/blazor/debug.md

```diff
@@ -45,54 +44,117 @@ Debugging requires either of the following browsers:
 * Google Chrome (version 70 or later)
 * Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com))
```

Comment:
The Chromium based Microsoft Edge is no longer in preview:  https://microsoft.com/edge. If you have Edge version 80 or later, you should be good to go.